apps/site/components/EOL/Modal.tsx

```diff
@@ -0,0 +1,156 @@
+                      </LinkWithArrow>
+                    </div>
+                  ))
+                : '-'}
+            </td>
+            <td>
+              <VulnerabilityChip severity={vuln.severity} />
+            </td>
+            <td className={classNames(maxWidth, 'truncate')}>
+              {vuln.description || vuln.overview || '-'}
+            </td>
+            <td>
+              {vuln.ref ? (
+                <LinkWithArrow
+                  href={vuln.ref}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {t('blogLinkText')}
+                </LinkWithArrow>
+              ) : (
+                '—'
+              )}
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  );
+};
+
+const UnknownSeveritySection: FC<{
+  vulnerabilities: Array<Vulnerability>;
+  hasKnownVulns: boolean;
+}> = ({ vulnerabilities, hasKnownVulns }) => {
+  const t = useTranslations('components.eolModal');
+
+  if (!vulnerabilities.length) {
+    return null;
+  }
+
+  return (
+    <details open={!hasKnownVulns}>
+      <summary className="cursor-pointer font-semibold">
+        {t('showUnknownSeverities')} ({vulnerabilities.length})
+      </summary>
+      <div className="mt-4">
+        <VulnerabilitiesTable
+          vulnerabilities={vulnerabilities}
+          maxWidth={'max-w-3xs'}
+        />
+      </div>
+    </details>
+  );
+};
+
+const EOLModal: FC<ModalProps> = ({ open, closeModal, data }) => {
+  const { release, vulnerabilities } = data as EOLModalData;
+  const t = useTranslations('components.eolModal');
+
+  const modalHeading = t(release.codename ? 'title' : 'titleWithoutCodename', {
+    version: release.major,
+    codename: release.codename ?? '',
+  });
+
+  const [knownVulns, unknownVulns] = vulnerabilities.reduce(
+    (acc, vuln) => {
+      acc[vuln.severity === 'unknown' ? 1 : 0].push(vuln as KnownVulnerability);
+      return acc;
+    },
+    [[], []] as [Array<KnownVulnerability>, Array<Vulnerability>]
+  );
+
+  knownVulns.sort(
+    (a, b) =>
+      SEVERITY_ORDER.indexOf(a.severity) - SEVERITY_ORDER.indexOf(b.severity)
+  );
+
+  const hasKnownVulns = knownVulns.length > 0;
+  const hasAnyVulns = hasKnownVulns || unknownVulns.length > 0;
+
+  return (
+    <Modal open={open} onOpenChange={closeModal}>
+      <Title>{modalHeading}</Title>
+      <Content>
+        {vulnerabilities.length > 0 && (
+          <p className="m-1">
+            {t('vulnerabilitiesMessage', { count: vulnerabilities.length })}
+          </p>
+        )}
+
+        {hasKnownVulns && <VulnerabilitiesTable vulnerabilities={knownVulns} />}
+
+        <UnknownSeveritySection
+          vulnerabilities={unknownVulns}
+          hasKnownVulns={hasKnownVulns}
+        />
+
+        {!hasAnyVulns && <p className="m-1">{t('noVulnerabilitiesMessage')}</p>}
+      </Content>
+    </Modal>
+  );
+};
+
+export default EOLModal;
```

Comment:
We have the tendency of naming the default export the same name as the file itself :)

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/7990/commits/07befbba1db1262e311b8364776f8ce1c6e5e182